packages/react-vapor/src/components/button/examples/ButtonExamples.1.use.md

```diff
@@ -0,0 +1,39 @@
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Primary   | Used to enable the most important actions, i.e., those that lead to the completion of the main task (e.g., Save, Continue, Submit), or strong calls to action. Ideally, there should be only one Primary Button per screen to avoid diluting user attention.                                                                                                                                                                                                |
+| Secondary | Used for main actions that users might want to execute but that are not in direct relation to the main task that is to be done in the screen or interface.                                                                                                                                                                                                                                                                                                           |
+| Combo     | Combo Buttons have an attached drop menu where a choice between multiple actions has to be made (2 or more). Those buttons are useful when one single action allows users to create, delete or modify many different elements, which can be selected in the dropdown list that appears upon clicking. While using a combo button prevents adding an extra step in the process, it can add friction to the user decision making process when the options in the dropdown list are not intuitive enough. |
+| Isolated  | Used for extra actions which are, as the name suggests, isolated in an interface. Those button do not have borders and background colors, in order to reduce noise and to make them less conspicuous. Icons are optional and may help users identify actions faster. However, keep in mind that too many icons can be as distracting and confusing as too few.                            |
+| Icon Only | Used when there is a lack of space and a high density of content. This button type is preferably used to represent labels that are easily represented by an icon, such as the word "More".                                                                                                                                                                                                                                                                           |
```

Comment:
‘easily represented by an icon, such as the word "More".' ? Do you mean that the word "More" is an icon? Or rather that it can easily be represented by an icon, such as "+" for instance? If I'm right, I would suggest:
that are easily represented by an icon, such as the word "More". -> that are easily represented iconically, such as the icon "+" for "more items".